package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "not-panda",
+  "version": "1.0.0",
+  "description": "An app that tells if you doodled a panda or not a panda",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/chenlwilson/not-panda.git"
+  },
+  "keywords": [
+    "panda",
+    "magenta",
+    "tensorflow",
+    "doodle"
+  ],
+  "author": "Chen Wilson",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/chenlwilson/not-panda/issues"
+  },
+  "homepage": "https://github.com/chenlwilson/not-panda#readme",
+  "scripts": {
+    "start": "nodemon server.js",
+    "react-dev": "webpack -d --watch"
+  },
+  "engines": {
+    "node": ">=4.0.0"
```

Comment:
you're using es6 modules (import x from y) instead of var x = require(y);

That's not supported until Node 8. For engines when you're not sure put in what you're using (which should be the latest LTS, for node that's node 10)